dask/array/tests/test_array_core.py

```diff
@@ -19,6 +20,7 @@
 
 import dask
 import dask.array as da
+import dask.dataframe
```

Comment:
Dask array tests should be runnable without pandas installed.  (also, note like 7 above, we even use `pytest.importorskip` with numpy.